src/Api/Custom.php

```diff
@@ -32,8 +34,8 @@ public function __construct(string $endpoint, Magento $magento)
      */
     public function __call($method, $args)
     {
-        if ($method == 'get' || $method == 'post') {
-            $args[0] = rtrim($this->endpoint, '/').'/'.$args[0];
+        if (in_array($method, self::HTTP_METHODS)) {
+            $args[0] = $this->endpoint.ltrim($args[0], '/');
```

Comment:
What would be the use of these? Can't we just do:

`$magento->api('/my-custom-endpoint')->delete('my-endpoint/12');`
`$magento->api('/my-custom-endpoint')->get('my-endpoint/12');`

Reply:
Just to make sure that we have a nice clean url without double slashes.

`$magento->api('/my-custom-endpoint')`
and
`$magento->api('my-custom-endpoint')`

should work